packages/xo-server/src/index.js

```diff
@@ -1,9 +1,11 @@
 import appConf from 'app-conf'
 import assert from 'assert'
+import authenticator from 'otplib/authenticator'
```

Comment:
Why not import `optlib` directly?

Reply:
Because i only need `authenticator` and i have this error `Cannot read property 'authenticator' of undefined` if i import like this `import otplib from 'otplib'`. 
But i can import `import { authenticator } from 'otplib'` if you prefer.